src/__tests__/tailwind.test.tsx

```diff
@@ -293,12 +306,51 @@ describe("tw", () => {
     it("should render component as component in `$as` prop: test 2", () => {
         const Div = tw.div<{ $test1: string }>`
         text-black
-        ${(p: any) => (p.$test1 === "true" ? `bg-gray-500` : ``)}
+        ${(p) => (p.$test1 === "true" ? `bg-gray-500` : ``)}
         `
-        const Nav = tw.nav<{ $isVertical: boolean }>`flex h-20 w-full ${(p: any) =>
+        const Nav = tw.nav<{ $isVertical: boolean }>`flex h-20 w-full ${(p) =>
             p.$isVertical === true ? "flex-col h-full w-20" : ""}`
         const RedDiv = tw(Div)`bg-red-500`
         const { asFragment } = render(<RedDiv $as={Nav} $test1="true" $isVertical={false} />)
         expect(asFragment()).toMatchSnapshot()
     })
+    it("should not clear merged inheritedClasses without $as", () => {
```

Comment:
Can we extend (or create another) which asserts that inherited classes are also set if the parent has any props on it?

Something like:
```
const Base = tw.button<{ $active: boolean }>`
  ${p => (p.$active ? "text-white" : "text-black")}
`
const Child = tw(Base)`
  ${p => (p.$active ? "bg-blue-600" : "bg-white")}
`

const { asFragment, getByRole } = render(<Child $active={true} />);
expect(getByRole('button')).toHaveClass("text-white bg-blue-600");
expect(asFragment()).toMatchSnapshot();
```